pkg/monitor/monitor/monitor_manager.go

```diff
@@ -236,12 +236,12 @@ func (m *MonitorManager) syncTidbMonitorConfig(tc *v1alpha1.TidbCluster, monitor
 				continue
 			}
 			selector := labels.NewSelector().Add(*r1).Add(*r2)
-			tcList, err := m.deps.Clientset.PingcapV1alpha1().TidbClusters(tcRef.Namespace).List(metav1.ListOptions{LabelSelector: selector.String()})
+			tcList, err := m.deps.TiDBClusterLister.List(selector)
```

Comment:
Please also change the `Clientset` in L80.

Reply:
And here https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/tidbcluster_status_manager.go#L65